### DISCUSSION TITLE
I can’t set up the repository

### DISCUSSION THREAD

**Discussion by bdcco** [CollectionBuilder/collectionbuilder.github.io#59] [Q&A]
Hi, I’ve been trying to create a website with the CollectionBuilder-GH template. But when it comes to the Step 2 ‘Turn on the GitHub Pages option to make the repository produce a website’, I can’t find the source: master-branch option. How do I find it?

**Comment by evanwill** [comment]
@bdcco unfortunately some of the interface option on GitHub have changed since the CB-GH tutorial was recorded, so it gets a bit confusing. Also, the default source branch is generally called "main" now (not "master"), unless you manually changed it! 
The up to date instructions are in the docs: https://collectionbuilder.github.io/cb-docs/docs/deploy/gh-pages/
